source/bao_hyp/config.rst

```diff
@@ -65,41 +65,512 @@ describe the image running on the VM (see `Guest Image`_);
+
+- ``cpu_affinity`` [optional] - corresponds to a bitmap signaling the preferred physical CPUs
+  assigned to the VM. If this value is mutually exclusive for all the VMs, the physical CPUs
+  assigned to each VM follow the bitmap. Otherwise (in case of bit overlap or lack of affinity
+  definition), the CPU assignment is defined by the hypervisor;
+
 Coloring
 ********
 
+Cache coloring is a technique used to partition shared Last Level Cache (LLC) sets among different
+guests (i.e., VMs). The main goal is to minimize cache conflicts and enhance overall system
+performance by carefully assigning specific colors, representing cache sets, to different entities.
+Bao supports cache coloring which can be configured using the ``colors`` field within the
+``vm_config`` struct.
+
+- ``colors`` [optional] - corresponds to a bitmap for the assigned cache colors of the VM. This
+  value is truncated depending on the number of available colors calculated at run-time, i.e., its
+  platform-dependent. By default, the coloring mechanism is not active. For instance, the following
+  picture depicts a hypothetical setup with a 50/50 coloring scheme;
+
+.. figure:: img/llc-colors.svg
+    :align: center
+    :width: 100%
+    :name: llc-colors-fig
+
+|
+
+It's important to note that cache coloring relies on the careful assignment of colors to each VM.
+However, this mechanism may not take effect if the physical mapping feature is enabled for a
+specific memory region. Cache coloring exclusively operates in virtual memory systems, i.e.,
+systems featuring Memory Management Units (MMUs) for address translation.
+
 Shared Memory Configuration
 ---------------------------
+.. _Shared Memory Configuration:
+
+In Bao's configuration, you can set multiple shared memory regions (e.g., ``shmem0``, ``shmem1``,
+...) or none at all. An ID is assigned to each shared memory object. Later, this ID can be linked
+to an IPC in the multi-guest configuration.
+
+.. figure:: img/shmem.svg
+    :align: center
+    :name: shmem-fig
+
+|
+
+The configuration of shared memory is done by populating a struct called ``shmem``. The struct
+definition is presented bellow:
+
+.. code-block:: c
+
+    struct shmem {
+        size_t size;
+        bool place_phys;
+        union {
+            paddr_t base;
+            paddr_t phys;
+        };
+    };
+
+
+This struct contains the following parameters:
+
+- **size** [mandatory] - defines the total size of the shared memory. The size of the shared \
+  memory must always be multiples of 4Kb (0x1000);
+- **place_phys** [optional] - This flag determines the mapping of shared memory into virtual \
+  memory. When ``place_phys`` is set to ``false`` (default value), the base address is a guest \
+  (virtual) address. When set to ``true``, the guest address directly corresponds to the physical \
+  address (PA) of the shared memory.
+- **base / phys** [optional] - When ``place_phys`` is set to ``false``, the ``base`` parameter \
```

Comment:
```suggestion
- ``size`` [mandatory] - defines the total size of the shared memory. The size of the shared \
  memory must always be multiples of 4Kb (0x1000);
- ``place_phys`` [optional] - This flag determines the mapping of shared memory into virtual \
  memory. When ``place_phys`` is set to ``false`` (default value), the base address is a guest \
  (virtual) address. When set to ``true``, the guest address directly corresponds to the physical \
  address (PA) of the shared memory.
- ``base`` / ``phys`` [optional] - When ``place_phys`` is set to ``false``, the ``base`` parameter \
```

Reply:
Thanks! Fixed in https://github.com/bao-project/bao-docs/pull/78/commits/9744a25a97ea9dbb532bb1011ae33889d826cd17!